docs/further_reading.dialog_element.md

```diff
@@ -3,19 +3,33 @@ title: Dialog element
+:::
+
+## Remaining caveats
+
+Here are some of the remaining issues with the HTML `<dialog>` element:
+
+- Clicking outside the dialog does not close the dialog in some browsers such as Chrome.
+- The `::backdrop` pseudo-element only shows when programatically opening the dialog, not when using the `open` attribute.
 - Default styles are left to the browsers’ discretion and can be inconsistent.
-- The [alert dialog pattern](advanced.alert_dialog.md) (`role="alertdialog"`) simply does not work with the dialog element.
-- It still requires JavaScript anyway, so it’s not even 100% HTML.
-- [Read more about the shortcoming of the dialog element by Scott O'hara](https://www.scottohara.me/blog/2019/03/05/open-dialog.html).
+- The [alert dialog pattern](advanced.alert_dialog.md) (`role="alertdialog"`) does not work with the `<dialog>` element.
+- Using the `<dialog>` element still requires JavaScript (unlike `<details>` for instance).
+
+:::note  
+If you are aware of a downside or pitfall of the `<dialog>` element that was not mentioned here, please kindly [open an issue](https://github.com/KittyGiraudel/a11y-dialog/issues) so we can keep this document relevant and informative.  
+:::
+
+## Why a11y-dialog
 
-Additionally, there are still reasons to use a11y-dialog (or a similar library):
+This brings us to the reasons why one may stick to using a11y-dialog (or a similar library):
 
-- Not wanting to deal with browsers inconsistencies. It would be a bad reason to use a JavaScript library **if** `<dialog>` worked without JavaScript, which it doesn’t. So if one wants a dialog that looks the same everywhere, then a library like a11y-dialog remains the way to go.
-- Having to support old browsers without the `<dialog>` element. I guess a polyfill might be better in a case like that, but also maybe not. Might as well use the same library for everything.
+- Not wanting to deal with browsers inconsistencies. It would be a bad reason to use a JavaScript library **if** the `<dialog>` element worked without JavaScript, which it doesn’t. So if one wants a dialog that looks the same everywhere, then a library like a11y-dialog remains a way to go.
+- Having to support old browsers without support for the `<dialog>` element. I guess a polyfill might be better in a case like that, but also maybe not. Note that a11y-dialog v8 is geared towards modern browsers though.
 - Needing `alertdialog` support. There is no plan for the `<dialog>` element to support `role="alertdialog"` as of today, so if that’s a need, then a11y-dialog remains a good option.
-- Building more complex interactions relying on events. The native `<dialog>` element doesn’t come with an event API, so one cannot react to it being open or closed. 
+- Building more complex interactions relying on events. The native `<dialog>` element doesn’t come with an event API, so one cannot react to it being open or closed.
```

Comment:
```suggestion
- Building interactions that rely on events. The native `<dialog>` supports [`cancel` and `close` events](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement#events), but their behaviors  aren't reliable. The `<dialog>` does not support an `open` event at all.
```

Not sure what the right level of information is here. In my testing, it appears that the dialog fires the `close` and `cancel` events when the user cancels a modal with `Escape`, which is confusing. Meanwhile, the dialog fires the `close` event only when using a button in `<form method="dialog">`.

[The spec](https://html.spec.whatwg.org/multipage/interactive-elements.html#the-dialog-element) has a section about how UAs may create an interface for firing `cancel` event, but nothing about how   that fires `close` at the same time.

I made [a codepen](https://codepen.io/mxmason/pen/oNMyVPO) to demo listening to the dialog's native events, and the quirks I mention here.